chronoline/chronoline.js

```diff
@@ -86,7 +86,7 @@ function getLeft(elem){
 }
 
 function getEndDate(dateArray){
-    return dateArray[dateArray.length - 1];
+    return new Date(Date.parse(dateArray[dateArray.length - 1]));
```

Comment:
This line was causing an exception before I added the parse to the `events.dates` loop on line 292. Tests are passing without this change now so guess it is not necessary. Will remove it from next commit.